tensorflow_cloud/containerize.py

```diff
@@ -138,6 +140,18 @@ def _create_docker_file(self):
             ):
                 self.docker_base_image += "-gpu"
 
+        if not self._base_image_exist():
+            if "dev" in self.docker_base_image:
+                warnings.warn("Docker base image {} does not exist.".format(self.docker_base_image))
+                newtag = "nightly"
+                if self.docker_base_image.endswith("-gpu"):
+                    newtag += "-gpu"
+                self.docker_base_image = self.docker_base_image.split(":")[0] + ":" + newtag
+                warnings.warn("Using TF nightly build.")
+            else:
+                raise ValueError("Local TF version {} does not have available base image.".format(self.VERSION))
```

Comment:
"There is no docker base image corresponding to the local TF version: {}. Please provide `docker_base_image` or try with an other TF version."

Reply:
Changed.